components/Stands/StandView.tsx

```diff
@@ -0,0 +1,207 @@
+    width: 100%;
+    height: 400px;
+  }
+  @media only screen and (max-width: 767px) {
+    height: 250px;
+  }
+`;
+
+const QAView = styled.div`
+  height: 800px;
+  width: 30%;
+  background-color: #222;
+  color: white;
+  @media only screen and (max-width: 993px) {
+    width: 100%;
+    height: 300px;
+  }
+`;
+
+const BackLink = styled.a`
+  font-size: 25px;
+  cursor: pointer;
+`;
+
+const Back = (): JSX.Element => (
+  <Flex>
+    <FlexItem>
+      <Link href="/stands">
+        <BackLink>{'< Tilbake til stands'}</BackLink>
+      </Link>
+    </FlexItem>
+  </Flex>
+);
+
+const SubPage = ({
+  page,
+  stand,
+}: {
+  page: string;
+  stand: StandView_stand;
+}): JSX.Element => {
+  switch (page) {
+    case 'om':
+      return <AboutPage stand={stand} />;
+    case 'program':
+      return <ProgramPage stand={stand} />;
+    case 'joblistings':
+      return <JobListingsPage company={stand.company} />;
+    default:
+      return <div>Noe gikk galt. Forsøk å refresh siden</div>;
+  }
+};
+
+const Stand = ({ stand }: Props): JSX.Element => {
+  const commonItemObj = {
+    active: (key: string): boolean => currentPage === key,
+    onClick: (key: string): void => setCurrentPage(key),
+  };
+
+  const navBarItems: React.ComponentProps<typeof NavBar>['items'] = [
+    {
+      text: `Om ${stand.company.name}`,
+      key: 'om',
+      ...commonItemObj,
+    },
+    {
+      text: `Programmet til ${stand.company.name}`,
+      key: 'program',
+      ...commonItemObj,
+    },
+    {
+      text: `Jobb i ${stand.company.name}`,
+      key: 'joblistings',
+      ...commonItemObj,
+    },
+  ];
+
+  const [currentPage, setCurrentPage] = useState(navBarItems[0].key);
+
+  return (
+    <>
+      <BorderlessSection noPadding>
+        <Back />
+        <div>
+          <img
+            src={stand.company.logo || undefined}
+            style={{ display: 'block', margin: '25px 0 25px' }}
+            alt={`${stand.company.name} logo`}
+          />
+        </div>
+      </BorderlessSection>
+      <LiveContainer>
+        <Player>
+          {stand.livestreamUrl ? (
+            <iframe
+              title="livestreamEmbed"
+              src={`${stand.livestreamUrl}?autoplay=true`}
+              style={{ position: 'absolute', top: '0', left: '0' }}
+              allowFullScreen
+              frameBorder="no"
+              width="100%"
+              height="100%"
+              referrerPolicy="no-referrer-when-downgrade"
+            ></iframe>
+          ) : (
+            <h3>Ingen stream for øyeblikket</h3>
+          )}
+        </Player>
+        <QAView>
+          {stand.qaUrl ? (
+            <iframe
+              title="slidoEmbed"
+              src={stand.qaUrl}
+              height="100%"
+              width="100%"
+              frameBorder="0"
+            ></iframe>
+          ) : (
+            <h3>Ingen Q&A for øyeblikket</h3>
+          )}
+        </QAView>
+      </LiveContainer>
```

Comment:
I've extracted this as it's own component in #515

Reply:
Ok, If we want to merge this now, I'll get the changes from that commit(s) into this branch so it's up to date.

Reply:
@olestole I've updated this with the changes you made. So we could rebase #515 with the changes implemented here I think.

Reply:
Alright, great!